apps/site/pages/en/blog/vulnerability/upcoming-cve-for-eol-versions.md

```diff
@@ -1,5 +1,5 @@
 ---
-date: '2025-01-06:00:00.000Z'
+date: '2025-01-06T16:00.000Z'
```

Comment:
@RafaelGSS was the goal within https://github.com/nodejs/nodejs.org/commit/8e5ef67e05ab7afd98d2448ff935065385e3800b midnight?

```suggestion
date: '2025-01-06T0:00.000Z'
```